src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -59,7 +59,7 @@ export class DotEditPageNavComponent implements OnInit {
 
         // Right now we only allowing users to edit layout, so no templates or advanced template can be edit from here.
         // https://github.com/dotCMS/core-web/pull/589
-        if (dotRenderedPage.layout) {
+        if (dotRenderedPage.template.drawed) {
```

Comment:
I remember that we decided to use layout instead of template because there was a case where this breaks because template is null or undefined, don't remember which case.